Tests/js/views/fields/assets/ez-author-editview-tests.js

```diff
@@ -372,100 +392,256 @@ YUI.add('ez-author-editview-tests', function (Y) {
 
             Y.Assert.isFalse(view.isValid(), "The input is not valid");
         },
+
+        "Should prefill with current user when new content": function () {
+            var view,
+                container;
+
+            Y.Mock.expect(this.content, {
+                method: 'isNew',
+                returns: true
+            });
+
+            view =  this.view = this._getView([], true);
+            container = this.view.get('container');
+            view.render();
+
+            this._assertNoNameError();
+            this._assertNoEmailError();
+            this._assertFieldError();
+            Y.Assert.areSame(container.one('.ez-field-author-email').get('value'),this.userEmail, 'Author mail should be the current user one' );
+            Y.Assert.areSame(container.one('.ez-field-author-name').get('value'), this.userName, 'Author mail should be the current user one' );
+            Y.Assert.isTrue(view.isValid(), "The input is valid");
+        },
     });
     Y.Test.Runner.setName("eZ Author Edit View tests");
     Y.Test.Runner.add(removeButtonTests);
     Y.Test.Runner.add(validationTests);
 
-    getFieldTest = new Y.Test.Case(
-        Y.merge(Y.eZ.Test.GetFieldTests, {
```

Comment:
you should keep that but make Y.eZ.Test.GetFieldTests a bit more flexible on the initialization of the tested view, that would be better than 2 big copy paste.